client/utils.go

```diff
@@ -0,0 +1,12 @@
+package client
+
+// Variable spec
+type Variable struct {
```

Comment:
maybe we should rename this to PipelineVariable?

Reply:
it is used by both pipeline and project
```
	Variables   []Variable `json:"variables,omitempty"`
```